src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -254,18 +254,29 @@ Transactor::checkSeqProxy(
     SeqProxy const t_seqProx = tx.getSeqProxy();
```

Comment:
Should we add a check for a `sfTicketSequence` field with the wrong type (i.e. a string?).

Reply:
Great question!

I had forgotten the details myself, so I did some poking.  The `sfTicketSequence` is an `SField` of type `uint32`.  Any time someone tries to serialize something other than a `uint32` into a `sfTicketSequence` then the serialization machinery generates an error.  By the time an `sfTicketSequence` has arrived at the transaction machinery the type of the data is already validated.

So, long story short, this kind of problem is being handled by our serialization machinery.  We don't need to do any validation here.

FWIW, I tried to add a quick unit test for this.  That doesn't work because the unit test framework itself uses the serialization machinery.  I can't even (easily) push this check through our unit tests.